pkgs/applications/misc/monero/default.nix

```diff
@@ -30,7 +30,6 @@ stdenv.mkDerivation {
         bin/monerod \
         bin/monero-blockchain-export \
         bin/monero-blockchain-import \
-        bin/monero-utils-deserialize \
```

Comment:
Just to justify removing this line, `monero-utils-deserialize` isn't built by default after monero-project/monero@71e2876033e0a5a814f464cad4b1f2459f3ec395.